ME-MYSELF-I/Weekly-meetings.md

```diff
@@ -1,6 +1,61 @@
 # Weekly Meetings
 
 * [15th September 2017](#date-15th-september-2017)
+* [24th September 2017](#date-15th-september-2017)
+
+
+### Date: 24th September 2017
+
+#### Who did you work directly with this week?
+
+* I was running CWL examples and discussing the problems with Jakub.
+
+#### Who did you help this week?
+
+* I had a short meeting with Alex
+
+#### Who helped you this week?
+
+* Yarik helped me with creating datalad repository from files saved in dropbox.
+* Chris and Oscar helped me with PR
+* Satra explained some issues regarding regression testing
+
+#### What did you achieve?
+
+* rerun all examples from nipype_tutorial and adding to datalad repo (it's very inefficient, still have to improve the workflow)
+* finished PR on networkx
+* run CWL tutorial 
+* solved an issue
+
+#### What did you struggle with?
+
+* I'm still not sure hot to efficiently create datalad repository from files from dropbox. Yarik corrected my files names, but I'm not sure what is the best way of using `git annex addurl` (more in my email).
```

Comment:
first something like this:

```python
def get_paths(root):
    paths = []
    values = dbx.files_list_folder(root)
    for entry in values.entries:
        if hasattr(entry, 'size'):
            paths.append(entry.path_lower)
        else:
            paths.extend(get_paths(entry.path_lower))
    return paths

foo = get_paths('/path/on/dropbox')
len(foo)

data = []
for path in foo:
    res = dbx.sharing_create_shared_link(path)
    data.append([path, res.url])

import json
with open('urls.json', 'wt') as fp:
    json.dump(data, fp, indent=2)
```

Reply:
Thanks, but I checked your code, and it looks like it does the same as my code, I'm creating the same list doing:

```
dbx = Dropbox(auth_code)

res = dbx.files_list_folder(path='/ds000114-nipype_tutorial_output', recursive=True)
cur = res.cursor
res_cd = dbx.files_list_folder_continue(cur)

with open('url_dj.txt', 'w') as file:
    for en in res_cd.entries:
        if hasattr(en, 'size'):
            file.write(en.path_lower + ", "  
                           + dbx.sharing_create_shared_link(en.path_lower).url.replace("www", "dl")      
                            + "\n")
```

Reply:
i'm just changing  `www` to `dl`

Reply:
Still don't work with `git annex` in a way I'm trying to use it: `git annex addurl --file <line.split(',')[0]> <ll.split(',')[1]>`

Reply:
ok, I was reading my files and found one more problem.... it might work now...

Reply:
Ok, so my problems came from the `entry.path_lower`. I didn't notice that it has always `/`, `git annex` was giving me only info " No such file or directory", so I thought that I have to create everything.

Everything works now and you can install from github: https://github.com/djarecka/datalad_repo_test

I can change names and I'm not sure if you want to connect it somehow to the main repo, or just keep it separately (so people don't download everything by running random commands).